conf/application.conf

```diff
@@ -219,11 +219,12 @@ nsi.get-account {
 
 # let min-backoff be `m` and max-backoff be `M`. Then `2m-M` cannot equal zero
 uc-threshold {
-  # this is a temporary flag to disable the DES theshold behaviour until we are ready in productino
-  enabled     = true
+  # this is a temporary flag to disable the DES theshold behaviour until we are ready in production
+  enabled     = false
```

Comment:
@MarcusQA is it more convenient to have the new threshold retrieval behaviour enabled or disabled by default?